src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1981,6 +1981,7 @@ internal enum ErrorCode
         ERR_FileScopedAndNormalNamespace = 8955,
         ERR_FileScopedNamespaceNotBeforeAllMembers = 8956,
         ERR_NoImplicitConvTargetTypedConditional = 8957,
+        ERR_NewlinesAreNotAllowedInsideANonVerbatimInterpolatedString = 8958,
```

Comment:
FYI, there will shortly be a merge conflict with PR [https://github.com/dotnet/roslyn/pull/55042](https://github.com/dotnet/roslyn/pull/55042)
The merging of a C# 10 feature will take precedence. So I'll let you deal with the conflict. Sorry about that

Reply:
i originally started with a gap, since this woudl be a c#11 feature.  i was asked to get rid of that gap :)  lmk what i should do here.

Reply:
This is a bugfix level change. It will ship in dev17, since the PR is targeting the `main` branch.
So it should be fine to use the next available error code (after I merge parameterless struct constructors). No need to use a new range.